apps/web/src/components/FiatOnRampModal/FiatOnRampModal.tsx

```diff
@@ -10,15 +10,16 @@ import {
   ModalWrapper,
   Text,
   useModal,
-  Liquidity,
 } from '@pancakeswap/uikit'
+import { LoadingDot } from '@pancakeswap/uikit/widgets/Liquidity'
+
 import { CommitButton } from 'components/CommitButton'
-import { MERCURYO_WIDGET_ID, MOONPAY_SIGN_URL, ONRAMP_API_BASE_URL } from 'config/constants/endpoints'
+import { MERCURYO_WIDGET_ID, MERCURYO_WIDGET_URL, ONRAMP_API_BASE_URL } from 'config/constants/endpoints'
+import crypto from 'crypto'
```

Comment:
I think this package is deprecated?

Reply:
should be a built-in nodejs module. If we're using on fe we need to find something else. Let's use nanoid if you want to generate random string?